packages/cadl-ranch-specs/http/dictionary/mockapi.ts

```diff
@@ -22,7 +22,7 @@ function createModelMockApis(route: string, value: any): MockApiGetPut {
       };
     }),
     put: mockapi.put(url, (req) => {
-      req.expect.bodyEquals(value);
+      req.expect.coercedBodyEquals(value);
```

Comment:
Please explain what you are fixing, your PR has no context

Reply:
`bodyEquals` just compares the values by taking them as literal strings. But if the value is datetime, this does not make sense, because the datetime could have different formats. Here `coerceBodyEquals` compares datetimes by first changing them into ISO format. We have same fix in the test server.

And we already have several fixes before.
This is for first solution: https://github.com/Azure/cadl-ranch/pull/91, https://github.com/Azure/cadl-ranch/pull/140
This is for second solution: https://github.com/Azure/cadl-ranch/pull/211

I just thought @timotheeguerin and @iscai-msft fully understand what I'm doing here, so I didn't put the description. :) Updated now.